src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -36,14 +36,37 @@
   </Node>
 
   <!--
-  Bound node that represents the binding of an "= Value" construct in a declaration.
+  Bound node that represents the binding of an "= Value" construct in a field declaration.
   Appears only in bound trees generated by a SemanticModel.
   -->
-  <Node Name="BoundEqualsValue" Base="BoundNode">
-    <!-- Expression representing the value. -->
-    <Field Name="Value" Type="BoundExpression"/>
+  <Node Name="BoundFieldEqualsValue" Base="BoundEqualsValue">
+    <!-- Field receiving the value. -->
+    <Field Name="Field" Type="FieldSymbol"/>
   </Node>
 
+  <!--
+  Bound node that represents the binding of an "= Value" construct in a property declaration.
+  Appears only in bound trees generated by a SemanticModel.
+  -->
+  <Node Name="BoundPropertyEqualsValue" Base="BoundEqualsValue">
+    <!-- Property receiving the value. -->
+    <Field Name="Property" Type="PropertySymbol"/>
+  </Node>
+    
+  <!--
+  Bound node that represents the binding of an "= Value" construct in a parameter declaration.
+  Appears only in bound trees generated by a SemanticModel.
+  -->
+  <Node Name="BoundParameterEqualsValue" Base="BoundEqualsValue">
+    <!-- Parameter receiving the value. -->
+    <Field Name="Parameter" Type="ParameterSymbol"/>
+  </Node>
+
+  <AbstractNode Name="BoundEqualsValue" Base="BoundNode">
```

Comment:
Will change. Is this an implied endorsement of having three derived types (for field, property, and parameter declarations) rather than one type that covers all of them?